saleor/graphql/order/tests/mutations/test_draft_order_create.py

```diff
@@ -115,6 +115,108 @@
     """
 
 
+DRAFT_ORDER_CREATE_MUTATION_VOUCHER_CODE = """
```

Comment:
I think this is not needed. We should use `DRAFT_ORDER_CREATE_MUTATION`

Reply:
done